tekton/release-cheat-sheet.md

```diff
@@ -130,6 +130,13 @@ the pipelines repo, a terminal window and a text editor.
 14. Announce the release in Slack channels #general and #pipelines.
 
 15. Update [the catalog repo](https://github.com/tektoncd/catalog) test infrastructure
-to use the new release by updating the `RELEASE_YAML` link in [e2e-tests.sh](https://github.com/tektoncd/catalog/blob/v1beta1/test/e2e-tests.sh).
+to use the new release by updating:
+* The `RELEASE_YAML` link in [e2e-tests.sh](https://github.com/tektoncd/catalog/blob/master/test/e2e-tests.sh).
+* The images used in catalog Tasks that are built and published with Pipelines:
```

Comment:
bump images but no bumping the version, for example, `task/git-clone/0.2` stays as is?

```
labels:
    app.kubernetes.io/version: "0.2"
```

Sorry for asking these questions, I am trying to understand the process. I understand we are still trying to find the right balance. 🙃

Reply:
yeah, as per [TEP-003](https://github.com/tektoncd/community/blob/master/teps/0003-tekton-catalog-organization.md#versioning-resources) we should have bumped the version of the resource but having different versions of the resource with a change in just the image version might not make sense until and unless there is a breaking change. Also in the current pipeline release there was some fix in `git-init` image which might have affected the users using the current `git-clone` task `v0.2` so that's why we decided to bump the image version of that task and not adding a new task version. Having said that, we will try to make sure that we bump the version of the task only if there is a breaking change and if there is a bug fix then we should update the current task version